dataeng/jobs/analytics/PrefectFlowsDeployment.groovy

```diff
@@ -0,0 +1,162 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_authorization
+import static org.edx.jenkins.dsl.AnalyticsConstants.slack_publisher
+
+
+class PrefectFlowsDeployment{
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job("prefect-flows-deployment-poll"){
+            authorization common_authorization(allVars)
+            logRotator common_log_rotator(allVars)
+            parameters secure_scm_parameters(allVars)
+            parameters {
+                stringParam('PREFECT_FLOWS_URL', allVars.get('PREFECT_FLOWS_URL'), 'URL for the prefect-flows repository.')
```

Comment:
No, I think I can remove this param

Reply:
We don't intend to use other than master when this will be productionize but as of starting weeks I think we should this parameter for debugging purpose